src/reader/widgets/ReaderWidget.vue

```diff
@@ -98,6 +98,14 @@
                   kind
                   urn
                   ref
+                  metricalAnnotations {
```

Comment:
as noted on ![](https://github.trello.services/images/mini-trello-icon.png) [Modularity of readers / reader components](https://trello.com/c/aDn12rrP), eventually we want leaner queries; a big benefit of GraphQL is that we can just declare and query for stuff we need when we need it (vs having to parse and carve up an entire `/text-parts` payload each time)